internal/api/handlers/scheduler_handler.go

```diff
@@ -79,3 +120,49 @@ func (h *SchedulerHandler) fromEntityToResponse(entity *entities.Scheduler) *api
 		Version: entity.Spec.Version,
 	}
 }
+
+func (h *SchedulerHandler) fromOperationsToResponses(entities []*operation.Operation) ([]*api.Operation, error) {
+	responses := make([]*api.Operation, len(entities))
+	for i, entity := range entities {
+		response, err := h.fromOperationToResponse(entity)
+		if err != nil {
+			return nil, err
+		}
+		responses[i] = response
+	}
+
+	return responses, nil
+}
+
+func (h *SchedulerHandler) fromOperationToResponse(entity *operation.Operation) (*api.Operation, error) {
+	status, err := fromOperationStatusToString(entity.Status)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert operation entity to response: %w", err)
+	}
+
+	return &api.Operation{
+		ID:             entity.ID,
+		Status:         status,
+		DefinitionName: entity.DefinitionName,
+		SchedulerName:  entity.SchedulerName,
+	}, nil
+}
+
+func fromOperationStatusToString(status operation.Status) (string, error) {
```

Comment:
Do you think it makes sense to move this function to something like `.String` for the `operation.Status`? Like what we have on [`GameServerStatus`](https://github.com/topfreegames/maestro/blob/next/internal/core/entities/game_room/game_room.go#L25).